docs/ecosystem-user-event.md

```diff
@@ -20,7 +20,7 @@ test('click', async () => {
   const { getByRole } = render(<textarea />)
 
   await userEvent.type(getByRole('textbox'), 'Hello, World!')
-  expect(getByRole('textbox')).toHaveAttribute('value', 'Hello, World!')
+  expect(getByRole('textbox')).toHaveProperty('value', 'Hello, World!')
```

Comment:
While we're here, should we use: https://github.com/testing-library/jest-dom#tohavevalue

```suggestion
  expect(getByRole('textbox')).toHaveValue('Hello, World!')
```

Reply:
Didn't realize that was a thing 😅  thanks